sysl2/sysl/seqs.go

```diff
@@ -130,38 +132,61 @@ func DoConstructSequenceDiagrams(
 }
 
 // DoGenerateSequenceDiagrams generate sequence diagrams for the given model
-func DoGenerateSequenceDiagrams(stdout, stderr io.Writer, flags *flag.FlagSet, args []string) {
+func DoGenerateSequenceDiagrams(args []string) error {
```

Comment:
Keep stdout, stderr io.Writer for now
there are pull requests in pipeline that will use these variables.

Reply:
I'll hold off merging this PR until these other PRs land, and I'll incorporate their changes. We do have to bite the bullet at some point, though.

Reply:
Per out of band discussion, I've temporarily reinstated the parameters.